doc/build.md

```diff
@@ -33,8 +33,8 @@ These options control some general build configuration values.
 | Option                      | Default | Description                                                                          |
 | ------                      | ------- | -----------                                                                          |
 | fast-strcmp                 | true    | Always optimize strcmp for performance (to make Dhrystone happy)                     |
-| have-alias-attribute        | auto    | Compiler supports __alias__ attribute (default autodetected)                         |
```

Comment:
These two changes look like you're re-naming these options and aren't just fixes?

Reply:
I thought it's a good idea to follow the meson style guide [when I change the option from combo type to feature type](https://github.com/picolibc/picolibc/pull/181/commits/fe3c58968a63a55eba0ef9dcef669a7bb6d8af1f). Of course I can keep the original name.  The option is used like:
```meson
if get_option('have-alias-attribute').enabled()
   do that
endif
```
https://mesonbuild.com/Style-guide.html#naming-options

Reply:
done